.github/workflows/release-python.yml

```diff
@@ -34,32 +27,73 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v3
 
-      - name: Build wheels on Linux
-        if: runner.os == 'Linux'
-        uses: pypa/cibuildwheel@v2.13.1
-        env:
-          # Linux cannot call "sh -c ./python/install-hyperonc.sh" because
-          # before-all command is also called with "sh -c"
-          CIBW_BEFORE_ALL: ./python/install-hyperonc.sh -u https://github.com/${{github.repository}}.git -r ${{github.ref}}
-        with:
-          package-dir: ./python
+      - run: |
+          echo "REF_NAME=${{github.ref_name}}" | tee -a $GITHUB_ENV
+          echo "EVENT_NAME=${{github.event_name}}" | tee -a $GITHUB_ENV
+          echo "PRERELEASE=${{github.event.release.prerelease}}" | tee -a $GITHUB_ENV
+          echo "TAG_NAME=${{github.event.release.tag_name}}" | tee -a $GITHUB_ENV
+          echo "COMMIT_HEAD=${{github.ref_name != '' && github.ref_name || env.GITHUB_SHA}}" | tee -a $GITHUB_ENV
 
-      - name: Build wheels on MacOSX
-        if: runner.os == 'macOS'
+      - name: Build wheels on ${{ matrix.os }}
         uses: pypa/cibuildwheel@v2.13.1
         env:
-          # Mac OS X cannot call "./python/install-hyperonc.sh" probably because
-          # "/bin/sh" is not correct path for the shell there
-          CIBW_BEFORE_ALL: sh -c "./python/install-hyperonc.sh -u https://github.com/${{github.repository}}.git -r ${{github.ref}}"
+          CIBW_BEFORE_ALL: sh -c "./python/install-hyperonc.sh -u https://github.com/${{github.repository}}.git -r ${{env.COMMIT_HEAD}}"
+          CIBW_SKIP: "*musllinux*"
```

Comment:
Not sure if we want to support musl? (I only heard of it recently) I feel that doing so would significantly increase build time, and there is not enough incentive to justify the effort. But yeah, there is no particular reason not to support that.
So I can add it back if you want.

Reply:
I thought about supporting `musl` to support wider range of platforms. Unfortunately we cannot distribute sources of our package because it requires Rust and C compiler to build the whole project. But I agree that `musl` is a rare animal and we can opt its support.

Reply:
I asked because I think you have some particular reason like build scripts incompatibility or something.

Reply:
Okay. I will opt out musl for now. If we need it back in the future, I can easily turn it on.